### PR TITLE
necessary changes to make ubuntu developer installation instructions work

### PR DESCRIPTION
**Pull request by tillmo**
@CarloDib has tried out the ubuntu developer installation and failed. This corrects some problems he encountered.

### CODE REVIEW

File: doc/developer_installation_on_ubuntu.md [hunk 1]
```diff
@@ -92,8 +86,16 @@ on the system, so we will have to install one:
 
 - `sudo apt-get install libqtwebkit-dev`
 
+Clone the ontohub sources:
+- `git clone git@github.com:ontohub/ontohub.git`
+Note that you need to get access rights from the Ontohub team first.
```
Comment (0robustus1) [inline doc/developer_installation_on_ubuntu.md]: Actually a clone via ssh/git still works if you do not have write-permissions on a repository. Only the push will later fail (as it should).

Comment (tillmo) [inline doc/developer_installation_on_ubuntu.md]: good point, the I will change this to ssh/git only.

**Comment by tillmo** [comment]
btw: do you know why postgres in installed from a ppa, and not from the regular Ubuntu sources?

**Comment by 0robustus1** [comment]
At the time of writing the version in the ubuntu sources did not contain features we were using. I'm not sure if that has changed.

Wrt. to the Pull Request: :+1:

**Pull request merged**